libseccomp/src/lib.rs

```diff
@@ -651,6 +651,198 @@ impl std::str::FromStr for ScmpArch {
     }
 }
 
+/// Represents a syscall number.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct ScmpSyscall {
+    nr: i32,
+}
+impl ScmpSyscall {
+    fn to_sys(self) -> i32 {
+        self.nr
+    }
+
+    fn from_sys(nr: i32) -> Self {
```

Comment:
Should be `pub`lic for `get_name*`.
Then we could also `impl From<i32>`. On the other hand could `ScmpSyscall::from(3)` mean anything.

Or just make `nr` `pub`? But `ScmpSyscall { nr: 3 }` is more verbose.